### PR TITLE
Delta uploads

### PR DESCRIPTION
**Pull request by darinkrauss**
@jh-bate

### CODE REVIEW

File: test/mongo/mongo.go [hunk 1]
```diff
@@ -46,7 +47,7 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 })
 
 func Address() string {
-	return "127.0.0.1"
+	return os.Getenv("TIDEPOOL_STORE_ADDRESSES")
```
Comment (jh-bate) [inline test/mongo/mongo.go]: so for the tests does this get set in the travis file?

Comment (darinkrauss) [reply test/mongo/mongo.go]: That environment variable is set in `env.test.sh` which is source'd in the Makefile which is invoked by Travis. So, we're all good here.

**Comment by darinkrauss** [comment]
Accidentally closed.

**Comment by darinkrauss** [comment]
[<M> DELTA UPLOADS (1 of 4) - backend support for delta uploads](https://trello.com/c/GPOsljJN/117-m-delta-uploads-1-of-4-backend-support-for-delta-uploads)

**Pull request closed without merging**